compiler/AST/iterator.cpp

```diff
@@ -1159,6 +1232,8 @@ void lowerIterator(FnSymbol* fn) {
 
   BlockStmt* singleLoop = NULL;
   if (CallExpr* singleLoopYield = isSingleLoopIterator(fn, asts))
+    // If the iterator contains a single loop statement containing a single
```

Comment:
I might move this comment above `BlockStmt* singleLoop = NULL;` or put braces around the if.  I think it's to easy to miss that there's no braces with the if when theres more than one "line" in the body.